templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} manufacturer:</h1>
+  <form action="" method="post">
+  {% csrf_token %}
+    {{ form|crispy }}
+    <input class="btn btn-primary" type="submit" value="{{ object|yesno:"Update,Create" }}">
```

Comment:
Similar to the heading, ensure that the `yesno` filter is applied to a boolean value to correctly set the submit button's value to 'Update' or 'Create'.